operate/qa/integration-tests/src/test/java/io/camunda/operate/util/OperateTester.java

```diff
@@ -254,20 +257,21 @@ public OperateTester createAndDeploySimpleProcess(String processId, String activ
     return this;
   }
 
-  public OperateTester deployProcess(String... classpathResources) {
-    Validate.notNull(zeebeClient, "ZeebeClient should be set.");
+  public OperateTester deployProcess(final String... classpathResources) {
+    Validate.notNull(zeebeClient, "CamundaClient should be set.");
```

Comment:
🔧 If we name this CamundaClient, we should also use it. Or stick with the Zeebe client for now.

Reply:
I think IntellJ modify those classes, so I'm going to rollback them